docs/api/1.x/_details-patch-object.rst

```diff
@@ -60,3 +53,38 @@ See :ref:`api-permissions-payload`.
 .. ----------------
 
 .. If a read-only field is modified, a |status-400| error is returned.
+
+JSON Patch Operations
+---------------------
+
+`JSON-Patch <https://tools.ietf.org/html/rfc6902>`_ is a way to define a sequence
+of operations to be applied on a JSON object.
+
+It's possible to use ``JSON-Patch`` notation setting ``Content-Type: application/jso-patch+json``.
```

Comment:
I'd probably use neither *emphasis* or `code` -- it's a proper noun, so capitals are enough.

I'd probably use `by` somewhere here, like:
```
It's possible to use JSON-Patch by sending the request header ``Content-Type: application/json-patch+json``.
```

I don't think it's necessary to explain "instead of (something else)".

Sorry to bikeshed :)